plugins/optimization-detective/load.php

```diff
@@ -127,5 +139,8 @@ class_alias( OD_URL_Metric_Group_Collection::class, 'OD_URL_Metrics_Group_Collec
 
 		// Add hooks for the above requires.
 		require_once __DIR__ . '/hooks.php';
+
+		// Load site health checks.
+		require_once __DIR__ . '/site-health/load.php';
```

Comment:
As noted below, I think the directory can be removed in favor of just a simpler `site-health.php` for now:

```suggestion
		require_once __DIR__ . '/site-health.php';
```